bundles/pascal/misc/example.pas

```diff
@@ -0,0 +1,53 @@
+{ Comments }
```

Comment:
These comments (at least) can be multi-line. Not covered in the example or lexer.